packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -332,6 +348,16 @@ class AndroidCameraCameraX extends CameraPlatform {
     processCameraProvider!.unbind(<UseCase>[preview!]);
   }
 
+  // Methods concerning camera information:
+
+  /// Adds fresh observers to the [LiveCameraState] of the current [camera].
+  Future<void> _updateLiveCameraState() async {
```

Comment:
Looking particularly for feedback on doing this in a helper that we'll want to call anytime `camera` is updated.